docusaurus/docs/guides/deployments/10-linux/20-router/60-backup.mdx

```diff
@@ -1,6 +1,6 @@
 ---
-title: Router Backup
-sidebar_label: Router
+title: Router Backup and Recovery
+sidebar_label: Backup
```

Comment:
same comment as controller backup/recovery

Reply:
I'll update the follow-on pull request branch based on whatever we decide here: https://github.com/openziti/ziti-doc/pull/867